src/binarySearch.js

```diff
@@ -0,0 +1,35 @@
+const binarySearch = (array , number) => {
+  let low = 0,
+      high = (array.length - 1),
+      mid
+
+  while ( array[mid] != number && low <= high ) {
```

Comment:
It's considered best practice to use strict equality/inequality checks (so that you check both type and value, instead of opening yourself up to potential bugs that arise from implicit type coercion):
```js
while( array[ mid ] !== number && low <= high ) {
```